src/control/server/config.go

```diff
@@ -69,6 +70,7 @@ type Configuration struct {
 	ControlLogJSON      bool                      `yaml:"control_log_json,omitempty"`
 	HelperLogFile       string                    `yaml:"helper_log_file"`
 	RecreateSuperblocks bool                      `yaml:"recreate_superblocks"`
+	ShutdownTimeout     string                    `yaml:"shutdown_timeout"`
```

Comment:
I think in certain situations we might want this to be user configurable, when for some reason or another we are expecting shutdown to take a longer time (system load or network delays for example). @ashleypittman thought it would be useful to have it user configurable, @johannlombardi thoughts? happy to remove from the PR if that's what we decide

Reply:
not currently advertised in the example configurations files on purpose, not part of normal configuration